rsconf/component/rsiviz.py

```diff
@@ -57,12 +57,15 @@ def internal_build_write(self):
             vhost=self.hdb.rsconf_db.host,
             j2_ctx=jc,
         )
+        e = PKDict(rsiviz=z, pykern=jc.pykern)
+        if "dice" in z:
+            e.dice = z.dice
         systemd.docker_unit_enable(
             self,
             jc,
             env=self.python_service_env(
-                PKDict(rsiviz=z, pykern=jc.pykern),
-                exclude_re=r"^rsiviz(?:__|_docker_image|_url_secret)",
+                e,
+                exclude_re=r"^rsiviz(?:__|_docker_image|_url_secret|_dice_)",
```

Comment:
Just a comment: Perhaps it should be include_re ?

Reply:
Possibly but as we add new modules to rsiviz and add configuration to them I think we'd have to update this. My hope when I originally added this block of code was we wouldn't have to change code as we add new configuration for rsiviz. Unless there is a regex that could cover it I'm not seeing?

Reply:
I don't think there's an easy answer. It depends on knowledge of the whole system.